### PR TITLE
add package_update_info as arg for user api

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies
thoth-station/adviser#1903

## This introduces a breaking change

- [x] Yes
- [ ] No

Depends on:
thoth-station/common#1171

## This Pull Request implements

add new param for indicating adviser is meant to update package

### CODE REVIEW

File: thoth/user_api/api_v1.py [hunk 1]
```diff
@@ -409,6 +409,7 @@ def post_advise_python(
     parameters["stack_info"] = parameters["input"].pop("stack_info", None)
     parameters["kebechet_metadata"] = parameters["input"].pop("kebechet_metadata", None)
     parameters["labels"] = parameters["input"].pop("labels", None)
+    parameters["package_update_info"] = parameters["input"].pop("package_update_info", None)
```
Comment (fridex) [inline thoth/user_api/api_v1.py:412]: Should the parameter be part of `_ADVISE_PROTECTED_FIELDS` to require a token for this input?

Comment (KPostOffice) [reply thoth/user_api/api_v1.py:412]: I don't think it needs to be. Users should be free to use this feature if they would like.

Comment (fridex) [reply thoth/user_api/api_v1.py:412]: Not sure about a use case when an external user would trigger an update of a package manually on user-api. I think this update should come from the system.

**Comment by goern** [comment]
Can we merge this?

**Pull request closed without merging**